ethereumj-core/src/main/java/org/ethereum/solidity/SolidityType.java

```diff
@@ -356,21 +357,15 @@ public Object decode(byte[] encoded, int offset) {
             return ByteUtil.bigIntegerToBytes(bi, 20);
         }
     }
-
-    public static class IntType extends SolidityType {
-        public IntType(String name) {
+    
+    public static abstract class NumericType extends SolidityType {
+        public NumericType(String name)
+        {
```

Comment:
Could you, please, move opening bracket `{` to the method definition line like in surrounding code in this all following cases.